__tests__/app.test.js

```diff
@@ -92,4 +94,34 @@ describe('/api/articles', () => {
             })
         })
     }); //errors???
+});
+
+describe('/api/articles/:article_id/comments', () => {
+    test('GET: responds with an array of comments for the given article', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(({body}) => {
+            expect(body.comments).toBeSortedBy('created_at')
+            body.comments.forEach((comment) => {
+                expect(comment.article_id).toBe(1)
+            })
+        })
+    });
+    test('GET: if passed an invalid number, responds with an error', () => {

```

Comment:
Only a tiiiny change here but I'd be specific about the article_id being invalid, rather than just a number